modules/criteoBidAdapter.js

```diff
@@ -35,22 +35,33 @@ var CriteoAdapter = function CriteoAdapter() {
     var biddingEventFunc = function () {
       var bids = params.bids || [];
 
+      var networkid;
+
       var slots = [];
 
       var isAudit = false;
 
       // build slots before sending one multi-slots bid request
       for (var i = 0; i < bids.length; i++) {
         var bid = bids[i];
+        var sizes = bid.sizes || [];
         slots.push(
           new Criteo.PubTag.DirectBidding.DirectBiddingSlot(
             bid.placementCode,
             bid.params.zoneId,
             undefined,
-            bid.transactionId
+            bid.transactionId,
+            sizes.map((size) => {
+              return { width: size[0], height: size[1] }
+              }
+            )
           )
         );
 
+        if (!networkid && bid.params.networkId != undefined) {
```

Comment:
Is there any way for `networkid` to be defined here? I don't see any assignments between here and its declaration.